tests/unit/crossplot/test_layout.py

```diff
@@ -0,0 +1,92 @@
+import os
+import time
+import dash
+from everviz.plugins._crossplot._layout._crossplot_layout import _get_radio_dropdown
+import pytest
+import dash_html_components as html
+import pandas as pd
+import everviz
+from everviz.plugins import Crossplot
+from PIL import Image
+
+
+@pytest.mark.parametrize(
+    "test_input,expected", [(["dropdown_val"], "dropdown_val"), (["1", "2"], "1")]
```

Comment:
Just a question, how does the test_input reflect the expected values here? I just don't see the logic behind the fact that a single element list should match a list of two elements, while a single element should give just one element?

I would expect: `"test_input,expected", [(["dropdown_val", "dropdown_val"], "dropdown_val"), (["1", "2"], "1")]`
to true..

Reply:
Might be poorly formulated, but the idea is to check that different types of input are valid, i.e. that expected value is selected from the dropdown, would this make it clearer:     
```"test_input,expected", [(["dropdown_val"], "dropdown_val"), (["1", "2"], "1"), (["1", "2"], "2")]```

Reply:
I mixed up what is `test_input` and what is `expected`, my bad.